src/content/getting-started/capz-cluster-creation/index.md

```diff
@@ -0,0 +1,425 @@
+          cidr: 10.223.3.128/26
+        hostCidr: 10.223.3.0/24
+        mode: private
+        podCidr: 192.168.0.0/16
+        serviceCidr: 172.31.0.0/16
+        workers:
+          cidr: 10.223.3.0/25
+    metadata:
+      description: bzm29 cluster
+      name: bzm29
+      organization: observability
+    providerSpecific:
+      location: westeurope
+      subscriptionId: 6b1f6e4a-6d0e-4aa4-9a5a-fbaca65a4711
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: bzm29-user-values
+  namespace: org-observability
+---
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+  name: bzm29
+  namespace: org-observability
+spec:
+  catalog: cluster
+  config:
+    configMap:
+      name: ""
+      namespace: ""
+    secret:
+      name: ""
+      namespace: ""
+  kubeConfig:
+    context:
+      name: ""
+    inCluster: true
+    secret:
+      name: ""
+      namespace: ""
+  name: cluster-azure
+  namespace: org-observability
+  userConfig:
+    configMap:
+      name: bzm29-user-values
+      namespace: org-observability
+  version: 0.0.17
+---
+apiVersion: v1
+data:
+  values: |
+    clusterName: bzm29
+    organization: observability
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: bzm29-default-apps-user-values
+  namespace: org-observability
+---
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+  name: bzm29-default-apps
+  namespace: org-observability
+spec:
+  catalog: cluster
+  config:
+    configMap:
+      name: ""
+      namespace: ""
+    secret:
+      name: ""
+      namespace: ""
+  kubeConfig:
+    context:
+      name: ""
+    inCluster: true
+    secret:
+      name: ""
+      namespace: ""
+  name: default-apps-azure
+  namespace: org-observability
+  userConfig:
+    configMap:
+      name: bzm29-default-apps-user-values
+      namespace: org-observability
+  version: 0.0.14
+```
+
+It will take a while once the workload cluster got provisioned. The current state can be seen by running `kubectl get cluster,azurecluster` against the management cluster:
+
+```nohighlight
+$ kubectl get cluster,azurecluster -o wide
+NAME                              PHASE         AGE   VERSION
+cluster.cluster.x-k8s.io/bzm29    Provisioned   9m
+
+NAME    CLUSTER   READY   REASON   MESSAGE   RESOURCE GROUP   SUBSCRIPTIONID                         LOCATION     ENDPOINT                                AGE
+bzm29   bzm29     True                       bzm29            6b1f6e4a-6d0e-4aa4-9a5a-fbaca65a4711   westeurope   apiserver.bzm29.azuretest.gigantic.io   8m50s
+```
+
+### login
+
+Once the cluster got created, `opsctl` could be used to login into the workload cluster by running
+
+```noghighlight
+$ opsctl login glippy jrp45
+```
+
+With the changed context, you are able to list the newly created nodes within a cluster:
+
+```noghighlight
+$ kubectl get nodes
```

Comment:
what do you prefer more - the short `k get nodes` output or the other one below where e.g. flatcar appears

Reply:
no is good